logger/src/main/java/com/orhanobut/logger/LogcatLogStrategy.java

```diff
@@ -1,11 +1,17 @@
 package com.orhanobut.logger;
 
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
 import android.util.Log;
 
+import static com.orhanobut.logger.Utils.checkNotNull;
+
 public class LogcatLogStrategy implements LogStrategy {
 
-  @Override public void log(int priority, String tag, String message) {
+  @Override public void log(int priority, @Nullable String tag, @NonNull String message) {
+    checkNotNull(message);
+
```

Comment:
In here, we can check empty/null case of tag and provide a default tag. "NO_TAG" would be ok.

Reply:
That'd be great if you also add a test to cover this case by confirming that we use "NO_TAG"

Reply:
Done